contracts/test/Tub.sol

```diff
@@ -14,6 +14,7 @@ contract Tub is Ownable {
     string private _value;
 
     event Store(string value);
+    event MultiStore(string indexed boxValue, address[] nested);
```

Comment:
question: is this testing multiple elements or complex elements?

_MultiStore is a bit vague (it could me simply many elements e.g. many strings), I think there may be a better description, depending on precisely what you're testing here_ 🤔

Reply:
it is to test filtering by an indexed string value
ok will rename